app/buyers/helpers/buyers_helpers.py

```diff
@@ -17,12 +17,12 @@ def get_framework_and_lot(framework_slug, lot_slug, data_api_client, allowed_sta
     return framework, lot
 
 
-def is_brief_correct(brief, framework_slug, lot_slug, current_user_id):
+def is_brief_correct(brief, framework_slug, lot_slug, current_user_id, allow_withdrawn=False):
```

Comment:
Ah yeah, was thinking of doing that but thought it would get a bit gnarly with having to string check for `'withdrawn'` status to avoid doing the `allow_withdrawn` check.